holodex.go

```diff
@@ -62,12 +62,14 @@ func QueueUpcomingStreams(app *Application) {
 
 			// Video already exists in db, skip it
 			if err == nil {
+				tx.Rollback()
```

Comment:
within `holodex.go` we do not use `defer tx.Rollback()` but instead call rollbacks manually before `continue`ing as `defer`ing within a `for` loop would leak resources.

we just need to take extra care when modifying this file in the future to also call `tx.Rollback()` when continuing the loop early (before `tx.Commit`)